### PR TITLE
Clarify "removing secrets from previous commits" section

### PR DESCRIPTION
**Pull request by lacymorrow**
### Why:

I've run into this issue multiple times. 

In the sections to "Removing a secret introduced by...", following the steps as described _will not_ do anything. Git is used by folks who are not get experts, therefore we should describe each step fully.

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Review by gtothe7 — COMMENTED** [review]
**1. > patch -`##> Languages   > ** [`新增列表（B_d，t，d）**`](ur)  ``(B_ 新增列表 `d, t,### **  > Languages   [Set (irl ** ms-outlook://people/v2/8df74a8b8a5671978a1b53ee333e1fdc8f67de6bba87177b194011f900f21eb16c90ade7899fbe038** 

👍🏻 # 1`

File: content/code-security/secret-scanning/working-with-secret-scanning-and-push-protection/working-with-push-protection-from-the-command-line.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ You can also remove the secret if the secret appears in an earlier commit in the
 
 1. Save and close the editor to start the interactive rebase.
 1. Remove the secret from your code.
+1. Stage your changes using `git add .`
```
Comment (mchammer01) [inline content/code-security/secret-scanning/working-with-secret-scanning-and-push-protection/working-with-push-protection-from-the-command-line.md:110]:
```suggestion
1. Add your changes to the staging area using `git add .`
```

**Review by mchammer01 — DISMISSED** [review]
@lacymorrow - thanks so much for spotting these inaccuracies and omissions ✨ 
Made a minor change to do with wording that I am going to apply.

**Review by mchammer01 — APPROVED** [review]
LGTM :shipit:

**Comment by lacymorrow** [comment]
Thanks folks! I ran into the issue again today 😅

**Comment by mchammer01** [comment]
What do you mean @lacymorrow? Your updates are in the docs. Do these need further tweaking?

**Comment by nguyenalex836** [comment]
I think @lacymorrow was expressing gratitude for the update to the docs, as she ran into a scenario again where clarity on these steps helped her 💛 But let me know if I am off!

**Comment by lacymorrow** [comment]
Exactly @nguyenalex836, I accidentally committed a secret and was pleasantly surprised to see that the docs had been updated with the complete steps :)

I am expressing gratitude that my PR was merged, the steps are now copy-pastable

**Pull request merged**